addons/point_of_sale/static/src/js/models.js

```diff
@@ -1382,6 +1410,26 @@ exports.PosModel = Backbone.Model.extend({
     getCurrencySymbol() {
         return this.currency ? this.currency.symbol : '$';
     },
+     /**
+     * Make the products corresponding to the given ids to be available_in_pos and
+     * fetch them to be added on the loaded products.
+     */
+    async _addProducts(ids){
+        await this.rpc({
```

Comment:
why you make a write here ?

If you want reprint a ticket with product not available, it set product available.